### ISSUE TITLE
Accessibility: ipython news page

### ISSUE THREAD

**Issue by Raymond-techy** [jupyterhub/outreachy#68]
### What page is this for?

https://ipython.org/news.html

### WAVE accessibility report

1 X Language missing or invalid
197 X Very low contrast
157 X Justified text
9 X Suspicious link text
![jupyter2](https://user-images.githubusercontent.com/81036251/196055385-95b1a16c-f7a9-41c2-a7d3-41cc5b294662.PNG)

### The HTML element to be changed

```html
<html xmlns="http://www.w3.org/1999/xhtml">

<a href="https://jupyter.readthedocs.io/en/latest/install.html" target="_blank" style="color: rgb(206, 92, 0); background-color: rgb(255, 255, 255);">
Install
</a>

<a href="documentation.html">
Documentation
</a>

<a href="project.html">
Project
</a>

<div class="section" id="ipython-7-7-and-above">
Heading level 2
<h2>
IPython 7.7 and above
Very low contrast
<a class="headerlink" href="#ipython-7-7-and-above" title="Permalink to this headline">
¶
</a>
</h2>
Unordered list
<ul class="simple">
<li>
<strong>
IPython 7.12.0
</strong>

: Released on Jan 31st 2020.
</li>
<li>
<strong>
IPython 7.11.0 and 7.11.1
</strong>

: Released on Dec 27, 2019 and Jan 1st 2020
</li>
<li>
<strong>
IPython 7.10.0 and 7.10.1
</strong>

: Released on Nov 27, 2019 and Dec 1st 2019
</li>
<li>
<strong>
IPython 7.9.0
</strong>

: Released on Oct 25, 2019
</li>
<li>
<strong>
IPython 7.8.0
</strong>

: Released on Aug 30, 1019
</li>
<li>
<strong>
IPython 7.7.0
</strong>

: Released on Jul 26, 2019
</li>
</ul>
Justified text
<p>
See the
Very low contrast
<a class="reference external" href="https://ipython.readthedocs.io/en/stable/whatsnew/version7.html">
release notes for version 7
</a>
</p>
</div>

<a class="reference external" href="http://channel9.msdn.com/shows/The+HPC+Show/Open-source-HPC-code-Episode-11-IPython-Grid-Engine-running-on-Windows-HPC-Server-2008/">
two
</a>
Very low contrast
<a class="reference external" href="http://channel9.msdn.com/shows/The+HPC+Show/Open-source-HPC-code-Episode-12-IPython-computes-150-million-digits-of-Pi-in-Parallel/">
videos
</a>

<a class="reference external" href="http://ipython.scipy.org/doc/nightly/html/parallel/parallel_winhpc.html">
here
</a>

<a class="headerlink" href="#what-will-become-ipython-0-11-is-taking-shape-big-changes-ahead" title="Permalink to this headline">
¶
</a>

<a href="http://sphinx-doc.org/">
Sphinx
</a>

<a class="reference external" href="https://ipython.org/download.html">
here
</a>


<img src="_static/nbviewer-thm-4.png" title="IPython Notebook Viewer">
</a>

<img title="Packt Publishing donates a portion of the proceeds from this book to support IPython's development." width="160" height="200" id="sidebar-book-cover" style="margin-bottom: 10px;" src="_static/jupyter-in-depth.png">

<a class="headerlink" href="#ipython-7-2" title="Permalink to this headline">
¶
</a>
```

### Your proposed HTML (or CSS), after change

```html
//apply the lang attribute 
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-us">

//adding different color to solve contrast issues 
<a href="https://jupyter.readthedocs.io/en/latest/install.html" target="_blank" style="color: rgb(206, 92, 0); background-color: rgb(255, 255, 255);">
Install
</a>

<a href="documentation.html" style="color:"#3333FF"; background-color: "#FFFFFF";">
Documentation
</a>

<a href="project.html" style="color:"#3333FF"; background-color: "#FFFFFF";">
Project
</a>

<div class="section" id="ipython-7-7-and-above">
Heading level 2
<h2>
IPython 7.7 and above
Very low contrast
<a class="headerlink" href="#ipython-7-7-and-above" title="Permalink to this headline" style="color:#3333FF; background-color: #FFFFFF;">
¶
</a>
</h2>
Unordered list
<ul class="simple">
<li>
<strong>
IPython 7.12.0
</strong>

: Released on Jan 31st 2020.
</li>
<li>
<strong>
IPython 7.11.0 and 7.11.1
</strong>

: Released on Dec 27, 2019 and Jan 1st 2020
</li>
<li>
<strong style="color:#3333FF; background-color: #FFFFFF;">
IPython 7.10.0 and 7.10.1
</strong>

: Released on Nov 27, 2019 and Dec 1st 2019
</li>
<li>
<strong>
IPython 7.9.0
</strong>

: Released on Oct 25, 2019
</li>
<li>
<strong>
IPython 7.8.0
</strong>

: Released on Aug 30, 1019
</li>
<li>
<strong>
IPython 7.7.0
</strong>

: Released on Jul 26, 2019
</li>
</ul>
Justified text
<p>
See the
Very low contrast
<a class="reference external" href="https://ipython.readthedocs.io/en/stable/whatsnew/version7.html" style="color:#3333FF; background-color: #FFFFFF;">
release notes for version 7
</a>
</p>
</div>

<a class="reference external" href="http://channel9.msdn.com/shows/The+HPC+Show/Open-source-HPC-code-Episode-11-IPython-Grid-Engine-running-on-Windows-HPC-Server-2008/" style="color:#3333FF; background-color: #FFFFFF;">
two
</a>
Very low contrast
<a class="reference external" href="http://channel9.msdn.com/shows/The+HPC+Show/Open-source-HPC-code-Episode-12-IPython-computes-150-million-digits-of-Pi-in-Parallel/" style="color:#3333FF; background-color: #FFFFFF;">
videos
</a>

<a class="reference external" href="http://ipython.scipy.org/doc/nightly/html/parallel/parallel_winhpc.html" style="color:#3333FF; background-color: #FFFFFF;">
here
</a>

<a class="headerlink" href="#what-will-become-ipython-0-11-is-taking-shape-big-changes-ahead" title="Permalink to this headline" style="color:#404040; background-color: #FFFFFF;">
¶
</a>

<a href="http://sphinx-doc.org/" style="color:#404040; background-color: #FFFFFF;">
Sphinx
</a>

<a class="reference external" href="https://ipython.org/download.html" style="color:#404040; background-color: #FFFFFF;">
here
</a>

<img src="_static/nbviewer-thm-4.png" title="IPython Notebook Viewer" alt="IPython Notebook Viewer">

<img title="Packt Publishing donates a portion of the proceeds from this book to support IPython's development." width="160" height="200" id="sidebar-book-cover" style="margin-bottom: 10px;" src="_static/jupyter-in-depth.png" alt="Packt publishing">

<a class="headerlink" href="#ipython-7-2" title="Permalink to this headline" style="color:#404040; background-color: #FFFFFF;">
¶
</a>
```

### More info

The page requires right justification of text and proper placement of icons

**Comment by minrk** [comment]
Thank you for the suggestions!
